src/styles/getMuiTheme.js

```diff
@@ -4,6 +4,21 @@ import ColorManipulator from '../utils/color-manipulator';
 import autoPrefix from './auto-prefix';
 import lightBaseTheme from './baseThemes/lightBaseTheme';
 import zIndex from './zIndex';
+import {autoprefixer, rtl} from './transformers';
+import compose from 'lodash.flowright';
+import warning from 'warning';
+
+const CALLED_ONCE = Symbol('prepareStyles/calledOnce');
+
+const callOnce = (transformer) => (style) => {
```

Comment:
We could, but after thinking about it, I wondered if this is a good idea?

What would happen would be in development it would just return the style, and in production it would continue flipping/autoprefixing with no warning. I think having this check in production makes it more consistent in terms of what you see in the development build.

What do you think?